engine/src/contract_methods/connector/deposit_event.rs

```diff
@@ -103,10 +140,13 @@ impl FtTransferMessageData {
         let recipient_address = Address::decode(&address)
             .map_err(errors::ParseEventMessageError::EthAddressValidationError)?;
 
+        #[allow(deprecated)]
         Ok(Self {
-            relayer: relayer_account_id.clone(),
             recipient: recipient_address,
-            fee,
+            fee: Some(FtTransferFee {
```

Comment:
Should this function be refactored to use `fee: None`? It is used to pass on the fee from the deposit event logs, but maybe we should be ignoring the fee in those logs as well.